src/V3Options.cpp

```diff
@@ -1173,6 +1181,7 @@ void V3Options::parseOptsList(FileLine* fl, const string& optdir, int argc,
     DECL_OPTION("-dumpi-", CbPartialMatchVal, [this](const char* optp, const char* valp) {
         m_dumpLevel[optp] = std::atoi(valp);
     });
+    DECL_OPTION("-no-json-addrs", Set, &m_noJsonAddrs);
```

Comment:
> I think we should default this as on. 

Since `--dump-tree-addrids` works, I think it is superior default to `--no-json-addr` (well, it even makes it kind of redundant).

Reply:
I made `--dump-tree-addrids` a default.